### ISSUE TITLE
Distorted Symbola font rendering in IE for some font sizes

### ISSUE THREAD

**Issue by karoltarasiuk** [mathquill/mathquill#530]
I'm attaching 2 screenshots with the issue. I created a page (full HTML source below) generating a digit of 6 in different sizes, from 10px to 40px.

Chrome (no distortion):
![screen shot 2016-02-25 at 2 50 43 pm](https://cloud.githubusercontent.com/assets/1845230/13309282/853771d0-dbcf-11e5-9dae-c4a32df025cd.png)

IE Edge (visible distortion, especially for lower font sizes, worst is probably for 14px and 16px):
![screen shot 2016-02-25 at 2 50 30 pm](https://cloud.githubusercontent.com/assets/1845230/13309305/b79c7198-dbcf-11e5-9e20-57e00152906c.png)

The issue is also visible on IE10 & 11. From what I see those browsers load `Symbola.woff` file. We also observed this issue with EOT and TTF within our product, so it seems like it's a general issue with this font in IE.

Source (based on your `demo.html` page I've built this one):
```
<!DOCTYPE html>
<html>
<head>
<title>MathQuill - Font issue in IE</title>
<link rel="stylesheet" type="text/css" href="support/home.css">
<link rel="stylesheet" type="text/css" href="../build/mathquill.css">

<style type="text/css">
code span {
  font: 90% Verdana, sans-serif;
}
#codecogsimg {
  vertical-align: middle;
  border: none;
}
.mathquill-text-field {
  width: 100%;
}
#html-source {
  display: none;
  font-size: 90%;
  white-space: pre-wrap;
}
.mq-math-mode .mq-editable-field {
  min-width: 1cm;
}
</style>

</head>
<body>
    <div id="body">
        <p>Font issue in IE 10, 11 and Edge (possibly older too):</p>
        <p class="sixes"></p>
    </div>
<script type="text/javascript" src="support/jquery-1.7.2.js"></script>
<script type="text/javascript" src="../build/mathquill.js"></script>
<script type="text/javascript">
MQ = MathQuill.getInterface(MathQuill.getInterface.MAX);
$(function() {
    var i,
        sixes = '',
        start = 10,
        end = 40;
    for (i = start; i <= end; i++) {
        sixes += '<span style="font-size: ' + i + 'px" class="mathquill-math-field">6<\/span>';
    }
    $('.sixes').html(sixes);
    $('.mathquill-math-field').each(function() { MQ.MathField(this); });
});
</script>
</body>
</html>

```

**Comment by michaelsharman** [comment]
Hi all, can I please ask for an ETA on this issue as we have clients who consider this urgent and we need to communicate if/when a fix will be forthcoming.

**Comment by laughinghan** [comment]
Hm, well as Karol says this is a general IE issue so what we can do may be limited. @karoltarasiuk What did you do about this with the other EOT and TTF fonts in your product?

Things I can try when I wake up tomorrow:
- ordering OTF before TTF
- generating the WOFFs from the OTF instead of the TTF (apparently WOFF is just a repackaging of fonts in one of those formats)
- as a workaround, using a CSS 2D transform to shrink the math field instead of reducing the font-size (e.g. `transform: scale(.8)`)

**Comment by karoltarasiuk** [comment]
Just to clarify it's a general issue in IE with this font, which is Symbola. I didn't recall any issues with other fonts, hence I reported it here. We could try with your suggestions @laughinghan. Workaround seems like could work but it's a last resort. We shouldn't expect people to use transform, as font should simply work. I think there maybe sth wrong with the glyph itself (vectors).

**Comment by andyclearnosity** [comment]
From my knowledge (fairly limited) this could be an issue with anti-aliasing and how IE renders and handles the pixels to display the typographic information.

I've seen CSS pieces that help with these issues, but looking at this from another angle, using these font's in anything below 13px will hurt accessibility anyway. So if it's possible, are we able to stop the font from going below a certain size i.e 13px

If not can we explore using CSS to help with the rendering of the font?

**Comment by andyclearnosity** [comment]
Wondering if this will help http://stackoverflow.com/questions/30876894/how-to-prevent-different-browsers-rendering-fonts-differently

**Comment by hiaik** [comment]
Hi @andyclearnosity 

Thanks for interesting readings! Unfortunately, we can not use CSS webkit as it is not supported in iOS and IE (http://caniuse.com/#search=-webkit-font-smoothing).

**Comment by laughinghan** [comment]
Good news, OTF indeed looks better than TTF, but unfortunately it also looks better than WOFF:
![capture](https://cloud.githubusercontent.com/assets/225809/14137394/7d0d0a12-f61c-11e5-85be-bfe6d52b7879.PNG)

Minimal diff that fixed the issue:
```diff
diff --git a/src/css/font.less b/src/css/font.less
index 06916b7..2330685 100644
--- a/src/css/font.less
+++ b/src/css/font.less
@@ -14,9 +14,9 @@
     local("Symbola Regular"),
     local("Symbola"),
     url(font/Symbola.woff2) format("woff2"),
-    url(font/Symbola.woff) format("woff"),
-    url(font/Symbola.ttf) format("truetype"),
     url(font/Symbola.otf) format("opentype"),
     url(font/Symbola.svg#Symbola) format("svg")
   ;
 }
```

This is unfortunate because while I see no issue with ordering the OTF before the TTF (they're pretty similar sizes), the WOFF is almost half the size of the OTF, and WOFF2 isn't yet supported on e.g. iOS where the size difference has the biggest effect.

I'm looking next into trying to regenerate the WOFF from the OTF, since I think it's just a compressed version of either; if we're lucky, we can get the WOFF to have the same better rendering that the OTF has.

**Comment by karoltarasiuk** [comment]
@laughinghan thanks for looking into it. Yeah - it's twice bigger, but I guess this will be a tradeoff we may have to accept. WOFF is a compressed OTF with some additional metadata, which could explain the loss of quality. We'll stay in touch - thanks!

**Comment by laughinghan** [comment]
From my understanding WOFF can be compressed OTF or compressed TTF&mdash;I'm suspecting that the WOFF we have right now is compressed TTF, and switching to a WOFF that is compressed OTF might alleviate this problem while continuing to be half the size (best of both worlds, essentially).

**Comment by karoltarasiuk** [comment]
@laughinghan Yes - that's correct of course. Will you be able to re-generate this font this way then?

**Comment by karoltarasiuk** [comment]
I've actually done it and added them here:
[fonts.zip](https://github.com/mathquill/mathquill/files/197100/fonts.zip)

![screen shot 2016-03-31 at 5 56 08 pm](https://cloud.githubusercontent.com/assets/1845230/14167877/ebe4d5ca-f769-11e5-98aa-1d14bcbb1ab9.png)

If all good PR is here: https://github.com/mathquill/mathquill/pull/598

**Comment by laughinghan** [comment]
Fixed by #598.

**Comment by laughinghan** [comment]
Actually IE8 still has this issue:
![internet_explorer_8_windows](https://cloud.githubusercontent.com/assets/225809/15593232/29c13728-235f-11e6-8a62-c89cfbbaa2b1.png)
https://circle-artifacts.com/gh/mathquill/mathquill/126/artifacts/0/tmp/circle-artifacts.RQJNfkV/imgs/Internet_Explorer_8_WINDOWS.png

(Thanks @mjdesa for the automated screenshots!)

Apparently IE8 only supports EOT: http://caniuse.com/font-face

So this can probably be fixed by re-generating the EOT from OTF rather than TTF, just like #598 did to fix the WOFFs?
